packages/babel-plugin-display-name/package.json

```diff
@@ -18,16 +18,17 @@
   "engines": {
     "node": ">=22.19.0"
   },
-  "scripts": {
-    "build": "tsc --project tsconfig.build.json"
-  },
   "dependencies": {
     "@babel/helper-plugin-utils": "^7.27.1",
     "@babel/core": "^7.28.5"
   },
   "devDependencies": {
+    "@sku-private/tsdown": "workspace:*",
     "@babel/preset-react": "^7.28.5",
     "@types/babel__core": "^7.20.5",
     "@types/babel__helper-plugin-utils": "^7.10.3"
-  }
+  },
+  "main": "./dist/plugin.mjs",
+  "module": "./dist/plugin.mjs",
+  "types": "./dist/plugin.d.mts"
```

Comment:
nit: can these be moved higher up in the file, e.g. above `exports`? Annoying that tsdown likes to just slap them at the bottom of the file 😠.